specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.generatedetailedcostreport.json

```diff
@@ -0,0 +1,373 @@
+          "type": "string"
+        },
+        "customerId": {
+          "description": "Customer Id for Modern (Invoice Id and billing profile is also required for this).",
+          "type": "string"
+        }
+      }
+    },
+    "GenerateDetailedCostReportTimePeriod": {
+      "description": "The start and end date for pulling data for the cost detailed report.",
+      "properties": {
+        "start": {
+          "description": "The start date to pull data from. example format 2020-03-15",
+          "type": "string"
+        },
+        "end": {
+          "description": "The end date to pull data to. example format 2020-03-15",
+          "type": "string"
+        }
+      },
+      "required": [
+        "start",
+        "end"
+      ]
+    },
+    "GenerateDetailedCostReportOperationStatuses": {
+      "description": "The status of the long running operation for cost detailed report.",
+      "properties": {
+        "id": {
+          "description": "The id of the long running operation.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of the long running operation.",
+          "type": "string"
+        },
+        "status": {
+          "description": "The status of the long running operation.",
+          "$ref": "#/definitions/Status"
+        },
+        "type": {
+          "description": "The type of the long running operation.",
+          "type": "string"
+        },
+        "error": {
+          "description": "The details of the error.",
+          "$ref": "common-types.json#/definitions/ErrorDetails"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The properties of the resource generated.",
+          "$ref": "#/definitions/DownloadURL"
+        }
+      }
+    },
+    "GenerateDetailedCostReportOperationResult": {
+      "description": "The result of the long running operation for cost detailed report.",
+      "properties": {
+        "id": {
+          "description": "The id of the long running operation.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of the long running operation.",
+          "type": "string"
+        },
+        "type": {
+          "description": "The type of the long running operation.",
+          "type": "string"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The properties of the resource generated.",
+          "$ref": "#/definitions/DownloadURL"
+        }
+      }
+    },
+    "DownloadURL": {
+      "description": "The URL to download the generated report.",
+      "properties": {
+        "downloadUrl": {
+          "description": "The URL to download the generated report.",
+          "type": "string"
```

Comment:
>type [](http://example.com/codeflow?start=11&length=4)

format uri

Reply:
This may get flagged as a breaking change, but will give it a try.

Reply:
This caused a breaking change as expected, I am reverting this.

Reply:
Fair enough